pandas/core/tools/datetimes.py

```diff
@@ -824,9 +824,6 @@ def to_datetime(
         out-of-bounds values will render the cache unusable and may slow down
         parsing.
 
-        .. versionchanged:: 0.25.0
-            changed default value from :const:`False` to :const:`True`.
-
```

Comment:
Removed this comment, since the description already covers that default is True.